src/data/staking-products.json

```
@@ -250,6 +279,32 @@
         "eventAction": "Clicked",
         "eventName": "Clicked ethdo go to link"
       }
+    },
+    {
+      "name": "AVADO",
```

```suggestion
      "name": "Avado",
```
Just changing this to be consistent